packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/pages/ListView/tests/ListView.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { Provider } from 'react-redux';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import userEvent from '@testing-library/user-event';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { MemoryRouter } from 'react-router-dom';
+
+import configureStore from '../../../../../../../../../admin/src/core/store/configureStore';
+import ReviewWorkflowsListView from '..';
+import { reducer } from '../../../reducer';
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  // eslint-disable-next-line react/prop-types
+  CheckPagePermissions({ children }) {
+    return children;
+  },
+}));
+
+const FIXTURE_WORKFLOW = {
+  id: 1,
+  stages: [
+    {
+      id: 1,
+      name: 'stage-1',
+    },
+  ],
+};
+
+const server = setupServer(
+  rest.get('*/review-workflows/workflows', (req, res, ctx) => {
+    return res(
+      ctx.json({
+        data: [FIXTURE_WORKFLOW],
+      })
+    );
+  })
+);
+
+const client = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+const ComponentFixture = () => {
+  const store = configureStore([], [reducer]);
+
+  return (
+    <MemoryRouter>
+      <QueryClientProvider client={client}>
+        <Provider store={store}>
+          <IntlProvider locale="en" messages={{}}>
+            <ThemeProvider theme={lightTheme}>
+              <ReviewWorkflowsListView />
+            </ThemeProvider>
+          </IntlProvider>
+        </Provider>
+      </QueryClientProvider>
+    </MemoryRouter>
+  );
+};
+
+const setup = (props) => {
+  return {
+    ...render(<ComponentFixture {...props} />),
+    user: userEvent.setup(),
+  };
+};
```

Comment:
Is there a reason you're not using the `wrapper` option? :) 
```suggestion
const ComponentFixture = () => <ReviewWorkflowsListView /> 

const setup = (props) => {
  return {
    ...render(<ComponentFixture {...props} />, {
     wrapper({ children }){
  const store = configureStore([], [reducer]);

  return (
    <MemoryRouter>
      <QueryClientProvider client={client}>
        <Provider store={store}>
          <IntlProvider locale="en" messages={{}}>
            <ThemeProvider theme={lightTheme}>
              {children}
            </ThemeProvider>
          </IntlProvider>
        </Provider>
      </QueryClientProvider>
    </MemoryRouter>
  );
}
    }),
    user: userEvent.setup(),
  };
};
```

Reply:
I've copied the test from `EditView` .. why not. Will update both 👍🏼

Reply:
Will help in the future when we have an abstraction around `render` with all the providers